### PR TITLE
Fix logger, tensorboard

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Fixes #607 to be able to do #609 

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

File: pytorch_lightning/logging/tensorboard.py [hunk 1]
```diff
@@ -59,23 +59,33 @@ def experiment(self):
...
             )
             return
-        self.experiment.add_hparams(hparam_dict=vars(params))
+        try:
+            # in case converting from namespace, todo: rather test if it is namespace
+            params = vars(params)
+        except TypeError:
+            pass
+        if params is not None:
+            self.experiment.add_hparams(hparam_dict=dict(params))
```
Comment (neggert) [inline pytorch_lightning/logging/tensorboard.py]: What problem is this fixing? The current implementation is consistent with all the other loggers.

**Comment by neggert** [comment]
Thanks for the bug fixes. I think maybe I did a rebase right at the end and forgot to re-run the tests. I promise they were all passing at some point.

File: pytorch_lightning/logging/tensorboard.py (hunk 1 above)
Comment (Borda) [reply pytorch_lightning/logging/tensorboard.py]: a dictionary cannot be converted via `vars`

**Comment by Borda** [comment]
well, not only, also `flush` is missing in older PT... :]

File: pytorch_lightning/logging/tensorboard.py (hunk 1 above)
Comment (neggert) [reply pytorch_lightning/logging/tensorboard.py]: Just for future reference, do we pass a dict anywhere? I thought we always passed some form of Namespace.

**Comment by neggert** [comment]
Yeah, missing the `flush` on Torch 1.1 was my mistake.

**Comment by Borda** [comment]
It is fixed, but more serious is missing `add_hparams` in 1.1 which means that they are not exported at all...

File: pytorch_lightning/logging/tensorboard.py (hunk 1 above)
Comment (Borda) [reply pytorch_lightning/logging/tensorboard.py]: In almost all tests... =)

**Pull request merged**